projects/services/public-api-service-typescript/src/service.ts

```diff
@@ -3,14 +3,21 @@ import bind from "bind-decorator";
 
 import { logger, config, topologyPeers, grpc, types } from "orbs-core-library/dist/common-library";
 import { topology } from "orbs-core-library/dist/common-library/topology";
-
+import { TransactionHandler, TransactionHandlerConfig } from "orbs-core-library/dist/public-api//transaction-handler";
```

Comment:
I'm removing the `dist` in my PR, but keep this in your code and I'll remove it while merging mine.